js/widgets/forms/flipswitch.js

```diff
@@ -251,9 +244,8 @@ return $.widget( "mobile.flipswitch", $.extend( {
 			var currentTheme = options.theme ? options.theme : "inherit",
 				newTheme = options.theme ? options.theme : "inherit";
 
-			this.widget()
-				.removeClass( "ui-bar-" + currentTheme )
-				.addClass( "ui-bar-" + newTheme );
+			this._removeClass( this.widget(), null,  "ui-bar-" + currentTheme );
+			this._addClass( this.widget(), null,  "ui-bar-" + newTheme);
```

Comment:
seeing this makes me thing we should add a `themePrefix` property to the theme extension and perhaps a themeElement prop too? 

@gabrielschulhof what are your thoughts here? Do we have many places where a sing widget applies theme classes to multiple elements where we may want to make this a hash even?